packages/web-twig/src/Resources/components/ButtonLink/ButtonLink.twig

```diff
@@ -13,6 +13,7 @@
 {# Class names #}
 {%- set _rootClassName = _spiritClassPrefix ~ 'Button' -%}
 {%- set _rootBlockClassName = _isBlock ? _spiritClassPrefix ~ 'Button--block' : null -%}
+{%- set _rootDisabledClassName = _isDisabled ? _spiritClassPrefix ~ 'Button--disabled' : null -%}
```

Comment:
Thanks for pointing this out! Can you please delete `_disabledAttr` and its usage too? Because `<a>` cannot have the `disabled` attribute.

Reply:
yes, deleted